client/my-sites/email/email-management/style.scss

```diff
@@ -266,7 +260,7 @@
 		}
 
 		> svg {
-			fill: white;
+			fill: #fff;
```

Comment:
Can you elaborate why do you prefer using not `#fff` over `#ffffff` and even over `white`?

Reply:
1) white vs #fff - the reason why I changed it was receiving lint error, no more :) 
2) #fff vs #ffffff - it's just a shortcut. It's very popular to use color shortcuts in CSS, since it's a bit easier to read and recognize white, black, and some gray colors, since you see just 3 symbols instead of 6 :) The second and more important point - it's A8C recommendation "[If you have to set a color, use lowercase hex values, and shorten them to their smallest expression (like #aaa).](https://github.com/Automattic/wp-calypso/blob/trunk/docs/coding-guidelines/css.md#the-dry-principle)"

Reply:
3. And I would say - it's not cool in CSS world to use named colors e.g. `white` 😁

Reply:
Is there any strict rule for colour codes in CSS? If not, let's not consider this a blocker for approving the PR, once the check for introductory offer is addressed :)

Reply:
@wongasy
1) As I mentioned before - I changed it since I received lint error and I could not push my main changes. Probably some rules are changed in configs.
2) Anyway, it's totally expected, since we have this rule in [coding guidelines](https://github.com/Automattic/wp-calypso/blob/trunk/docs/coding-guidelines/css.md#the-dry-principle).